agents.py

```
@@ -209,16 +201,23 @@ def ask_expert(question: str) -> str:
     return answer["retrievalResults"]
 
 
+from litellm import completion
```

Ruff

`litellm.completion` imported but unused

Remove unused import: `litellm.completion`

(F401)